docs/text_summaries.ipynb

```diff
@@ -0,0 +1,327 @@
+        "\n",
+        "from datetime import datetime\n",
+        "import json\n",
+        "from packaging import version\n",
+        "import tempfile\n",
+        "\n",
+        "print(\"TensorFlow version: \", tf.__version__)\n",
+        "assert version.parse(tf.__version__).release[0] >= 2, \\\n",
+        "    \"This notebook requires TensorFlow 2.0 or above.\""
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "qNsjMY0364j4"
+      },
+      "source": [
+        "## Logging a single piece of text\n",
+        "\n",
+        "To understand how the Image Summary API works, you're going to simply log a bit of text and see how it is presented in tensorboard.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "FxMPcdmvBn9t"
+      },
+      "outputs": [],
+      "source": [
+        "my_text = \"Hello world! 😃\""
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "IJNpyVyxbVtT"
+      },
+      "outputs": [],
+      "source": [
+        "# Clear out any prior log data.\n",
+        "!rm -rf logs\n",
+        "\n",
+        "# Sets up a timestamped log directory.\n",
+        "logdir = \"logs/text_basics/\" + datetime.now().strftime(\"%Y%m%d-%H%M%S\")\n",
+        "# Creates a file writer for the log directory.\n",
+        "file_writer = tf.summary.create_file_writer(logdir)\n",
+        "\n",
+        "# Using the file writer, log the text.\n",
+        "with file_writer.as_default():\n",
+        "  tf.summary.text(\"My first text\", my_text, step=0)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "rngALbRogXe6"
+      },
+      "source": [
+        "Now, use TensorBoard to examine the image. Wait a few seconds for the UI to spin up."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "T_X-wIy-lD9f"
+      },
+      "outputs": [],
+      "source": [
+        "%tensorboard --logdir logs"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "c8n8YqGlT3-c"
+      },
+      "source": [
+        "<!-- <img class=\"tfo-display-only-on-site\" src=\"https://github.com/tensorflow/tensorboard/blob/master/docs/images/images_single.png?raw=1\"/> -->"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "34enxJjjgWi7"
+      },
+      "source": [
+        "## Organizing multiple text streams\n",
+        "\n",
+        "If you have multiple streams of text, you can keep them in separate namespaces to help organize them, just like scalars or other data.\n",
+        "\n",
+        "Note that if you log text at many steps, TensorBoard will subsample the steps to display.  Again, this behavior is the same as for scalars, where the data is subsampled to make the presentation manageable."
```

Comment:
Thanks for the context.  Updated to mention the `--samples_per_plugin` and weaken the analogy to scalars.  Also updated the example to explicitly use the 'multiple_texts' subdir and to explicitly use the --samples_per_plugin flag, since the value (k/v as string) is not instantly obvious (I tried an integer first)